src/main/java/org/folio/inventory/domain/instances/Instance.java

```diff
@@ -76,6 +78,7 @@
   private String statusId;
   private String statusUpdatedDate;
   private Metadata metadata = null;
+  private List<String> tag;
```

Comment:
Is this a single tag, or is it intended to represent multiple tags?
```suggestion
  private List<String> tags;
```

Reply:
@marcjohnson-kint it is meant to be multiple, will fix it. Thanks